pkgs/servers/sozu/default.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sozu";
-  version = "0.11.56";
+  version = "0.11.59";
 
   src = fetchFromGitHub {
     owner = "sozu-proxy";
     repo = pname;
     rev = version;
-    sha256 = "sha256-/XyBzhZCsX9sGk+iTFlDnblWfDCZdI4b9yfo4Z+Wp1U=";
+    sha256 = "0khacsmpr54d18vmkrkqbbp2idf137gwjv3gsilxp9s76k4cjsv1";
   };
 
-  cargoSha256 = "sha256-F5EjBWHBNaQipxCi9Kiz5UTPMU0DuRf15NIVVvhiRxY=";
+  cargoSha256 = "1y5xxldjccdxn31b2wzdy71jh5kdxl3jwslf1haf83jicfvzanba";
 
   buildInputs =
     lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

Comment:
```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
```

Reply:
that `[ Security ]` makes the build fail, complaining about the variable `Security` not being defined.
importing it gives me this error:
```
error: anonymous function at /home/crns/code/repos/NixOS/nixpkgs/pkgs/servers/sozu/default.nix:1:1 called without required argument 'Security', at /home/crns/code/repos/NixOS/nixpkgs/lib/customisation.nix:69:16
```
what am I missing?

Reply:
You are missing https://www.github.com/NixOS/nixpkgs/blob/02a567192e752374c754302be82f9ea8d9b036a2/pkgs/top-level/all-packages.nix#L10579